src/main/java/com/slack/kaldb/chunk/ReadWriteChunkImpl.java

```diff
@@ -131,9 +132,11 @@ public void preSnapshot() {
   public boolean snapshotToS3(String bucket, String prefix, S3BlobFs s3BlobFs) {
     LOG.info("Started RW chunk snapshot to S3 {}", chunkInfo);
 
+    IndexCommit indexCommit = null;
```

Comment:
Null initialization is redundant and can be omitted.

Reply:
![image](https://user-images.githubusercontent.com/158041/121706583-9b9d3180-ca8a-11eb-86cb-e052f181fd51.png)

I can't remove the null check otherwise the compiler is complaining at me